Adding documentation to run development in Tina (#796)
tinacms/tinacms.org: content/docs/contributing/setting-up.md

npm run start
```

**Do not run** `npm install` from inside the `packages` directory

> **TinaCMS** uses [**Lerna**](https://lerna.js.org/) to manage dependencies when developing locally. This allows the various packages to reference each other via symlinks. Running `npm install` from within a package replaces the symlinks with references to the packages in the npm registry.

## Commands

| Commands | Description |
| --- | --- |
| npm run bootstrap | Install dependencies and link local packages. |
| npm run build | Build all packages |
| npm run test | Run tests for all packages |
| lerna run build --scope <package> | Build only <package>. |
| lerna run watch | Watch all packages for rebuilds. |

## Links

* [Circle CI](https://circleci.com/gh/tinacms/tinacms): Continuous Integration

## Run Development Packages 

> To contribute to Tina code, after installing the development setup we can run the Demo applications contained in the _packages_ directory, how **demo-cra**, **demo-gatsby** and **demo-next**.

Reviewer: ```suggestion
After installing the development setup you can run demo applications contained in the _packages_ directory **demo-cra**, **demo-gatsby** and **demo-next**.
```

> These projects can be used as a development environment for Tina packages, you can edit any other package contained within Tina packages how **@tinacms** and **react-tinacms-editor**.

Reviewer: ```suggestion
These projects can be used as a development environment for Tina packages, you can edit any other package contained within Tina packages.
```
